shortcodes.php

```diff
@@ -46,13 +46,13 @@ function ucf_people_list_shortcode( $atts, $content='' ) {
 
 		$exlude = array();
```

Comment:
Not related to what you've fixed, but I just noticed `$exclude` is misspelled here (`$exlude`).  Can you fix this typo while you have the project open?